mam/v2/mam/endpoint.c

```diff
@@ -50,3 +52,122 @@ void mam_endpoint_destroy(mam_ialloc_t const *const allocator,
 
   mam_mss_destroy(allocator, &endpoint->mss);
 }
+
+size_t mam_endpoint_serialized_size(mam_endpoint_t const *const endpoint) {
+  size_t mss_size = mss_stored_size(&endpoint->mss);
+
+  return pb3_sizeof_ntrytes(MAM2_ENDPOINT_ID_SIZE / 3) +
+         pb3_sizeof_size_t(mss_size) + pb3_sizeof_ntrytes(mss_size / 3);
```

Comment:
`mss_size` is in trits, but `pb3_sizeof_size_t` should get number of trytes i think